.github/workflows/publish-docs.yml

```diff
@@ -36,6 +36,19 @@ jobs:
       - name: Install Packages
         run: pnpm install --frozen-lockfile
 
+      - name: Update release verion
+        run: |
+          pnpm --filter mermaid run docs:release-version
+          pnpm --filter mermaid run docs:build
+
+      - name: Commit changes
+        uses: EndBug/add-and-commit@v9
+        with:
+          add: '["docs", "packages/mermaid/src/docs"]'
+          author_name: ${{ github.actor }}
+          author_email: ${{ github.actor }}@users.noreply.github.com
+          message: 'chore: Update MERMAID_RELEASE_VERSION in docs'
```

Comment:
I have a couple of issues with this block:

This code currently won't work without giving the `permission: contents: write`, and I'm very uncomfortable with giving this permission to any GitHub Action, since then it would be very easy for malicious code to commit malicious things to the Mermaid repo.

I would much rather prefer if we instead making a **RELEASE.md** file, and in it, we just ask the releaser to manually run these scripts before making the release commit.

If we do still decide to use a GitHub Action to automatically do this, then:

- If we do use `EndBug/add-and-commit`, instead of using `@v9`, we should probably use a Git hash like `@1bad3abcf0d6ec49a5857d124b0bfb52dc7bb081` instead, since that way, we'll only run a new version of this action when [@renovate[bot]](https://github.com/marketplace/renovate) makes a PR.
  - Or maybe we just use some `git commit` commands instead.
- I'd prefer if the author (or at least the commiter) was `41898282+github-actions[bot]@users.noreply.github.com`, so that we know that this was an auto-generated commit.

Reply:
Ok, how about a different approach?

We add a check in `release/*` branch to ensure there is no `MERMAID_RELEASE_VERSION` string present. 
So all release PRs will fail till someone runs the script manually.

Reply:
:+1: That's a great work around!

Maybe the check should also run in the `master` branch too.